builder/tart/diskutil_parser.go

```diff
@@ -0,0 +1,63 @@
+package tart
+
+import (
+	"fmt"
+	"howett.net/plist"
+)
+
+const expectedLastPartitionContent = "Apple_APFS"
+
+func ParseDiskUtilPlistOutput(input []byte) (string, string, error) {
```

Comment:
Don't we just need the dist disk in `AllDisks` or event `WholeDisks`?

Reply:
> Don't we just need the dist disk in `AllDisks` or event `WholeDisks`?

We do, but it would be also nice to make sure that the last partition is `Apple_APFS` and not `Apple_APFS_Recovery` to prevent mistakes in the future.

Reply:
Let's add a comment that explains what this method does and returns.

Reply:
See https://github.com/cirruslabs/packer-plugin-tart/pull/34/commits/ef64e36f8b65fd44537de44b5f806406263ee67a.